content/en/docs/toolbox/metering-and-billing/view-resource-consumption.md

```diff
@@ -27,17 +27,24 @@ KubeSphere metering helps you track resource consumption within a given cluster
 
    On the right side, there are three parts showing resource consumption in different ways.
 
-   ### Overview
-
-   The top part displays a consumption overview of different resources in a cluster since its creation. You can also see the billing information if you [have set prices for these resources](../enable-billing/) in the ConfigMap `kubesphere-config`.
-
-   ### Consumption by Yesterday
-
-   The middle part shows the total resource consumption by yesterday while you can also customize the time range and internal to see data within a specific period.
-
-   ### Current Resources Included
-
-   The bottom part displays the consumption of resources included in the selected target object (in this case, all nodes in the selected cluster) over the last hour.
+   <table>
+     <tbody>
+       <tr>
+         <th width='200'>Module</th>
+         <th>Description</th>
+       </tr>
+        <tr>
+         <td>Overview</td>
+          <td>Displays a consumption overview of different resources in a cluster since its creation. You can also see the billing information if you <a herf='../enable-billing/'>have set prices for these resources</a> in the ConfigMap <code>kubesphere-config</code>.</td>
+       </tr> <tr>
+         <td>Consumption by Yesterday</td>
+         <td>Displays the total resource consumption by yesterday. You can also customize the time range and internal to see data within a specific period.</td>
```

Comment:
I'm thinking whether we can use `Displays...` as the way written here. @Patrick-LuoYu What is your opinion?